kolibri/content/serializers.py

```diff
@@ -0,0 +1,20 @@
+from models import ContentMetadata, File, Format
+from rest_framework import serializers
+
+
+class ContentMetadataSerializer(serializers.ModelSerializer):
```

Comment:
How do we set the channel with this?

One appealing solution is the namespace the content metadata serializer by channel, so the endpoint url looks something like this:

`api/<channel_id>/content/<content_id>`

So all content endpoints could be namespaced by channel in the URL?